Dockerfile

```diff
@@ -1,34 +0,0 @@
-FROM ubuntu:18.04 as base
```

Comment:
I understand the other dockerfile is what we want for now. Can we rename that to just `Dockerfile` replacing this file?

Reply:
done